vysledek/index.html

```diff
@@ -0,0 +1,199 @@
+
+                    <p><strong>Reykjavík</strong> - Hlavní město určitě nevynecháte kvůli doplnění proviantu. Za zmínku stojí zajímavě architektonicky řešený kostel Hallgrímskirkja, centrum města a přístav se svou výstavbou začínají podobat všem vyspělým velkoměstům v Evropě a příliš milovníky severu neosloví. Zavítáte-li však do spleti uliček pod katedrálou, nebudete litovat, ty už atmosféru mají.</p>
+
+                    <p><strong>Gufossll</strong> - Vodopádové menu začínáme u kaskádovitého vodopádu na řece Hvítá. Postavit se na hranu mezi kaskádami a hledět do té masy vody, řítící se do kaňonu pod vámi je vážně zážitek&hellip;</p>
+
+                    <p><strong>Geysir</strong> - Kousek od Gullfossu je oblast, která turisty přitahuje jako magnet. Geysir, který dal název všem ostatním gejzírům světa. On sám se však již téměř odmlčel, údajně chrlí vodu jen velmi zřídka. Zato jeho menší a mladší soused, pojmenovaný Strokkur, prská do okolí vařící sloupec vody v méně či více pravidelných intervalech každých pár minut.</p>
+
+                    <p><strong>Seljalandsfoss</strong> - To už jsme na jižním pobřeží. Tento 60 metrů vysoký vodopád je výjimečný především tím, že se na něho můžete podívat nejen shora, ze stran, ale také jej obejít a hledět do okolní krajiny přes masy vody, které kolem Vás padají.</p>
+                </div>
+            </div>
+
+            <img src="/images/island3.jpg" alt="Island">
+            <p class="autor">Photo &copy; <a href="https://flic.kr/p/LfcKHP">Yiannis Chatzitheodorou</a></p>
+
+
+            <div class="text">
+                <p><strong>Skógafoss</strong> - Ať si koupíte jakoukoliv mapu nebo průvodce Islandu, jsem si téměř jist, že na titulce bude právě Skógafoss. Nutno říct, že právem. Výška 60 metrů, šířka 25 metrů, zážitek neuvěřitelný. Jakmile trochu zasvítí slunce, okamžitě se ve vodní tříšti v okolí vytváří krásná duha. Samozřejmě opět přitahuje spoustu turistů, takže doporučuji navštívit tuto hříčku přírody někdy pozdě večer či brzy ráno, kdy tu budete téměř sami, opravdu se to vyplatí.</p>
+
+                <p><strong>Dyrhólaey</strong> - Jde o nejjižnější cíp celého strova s vysokými útesy, na kterých ční jako tichý strážce velmi fotogenický maják. Zdejší útesy jsou také domovem Papuchalků (Fratercula arctica). Tito ptáci jsou opravdoví klauni a jejich nemotorný pohyb prostě musí vykouzlit úsměv na tváři každému. Mohutný lávový útes, ve kterém voda vyhloubila dvě skalní brány nebývalých rozměrů, prostě nesmíte vynechat.</p>
+            </div>
+
+
+            <div class="underline">
+                <h2><a name="galerie">Fotogalerie</a></h2>
+            </div>
+
+            <div class="gallery">
+
+
+                <div class="foto">
```

Comment:
Vlastně nevím, jestli jsi dělala hard nebo soft verzi, ale pokud hard, tak kdyžtak koukni do zadání soft, jak jsou tu použité sémantické značky `figure` a` figcaption`, nebo viz https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure